lib/app/AttachedExtensionsWatcher.js

```diff
@@ -21,21 +21,33 @@ class AttachedExtensionsWatcher extends EventEmitter {
       fsEx.readJson(path, (err, config) => {
         if (err) throw err
 
-        const updatedExtensions = Object.keys(config.attachedExtensions)
+        const updatedExtensionsKeys = Object.keys(config.attachedExtensions)
```

Comment:
way to complicated for no obvious gain, imho the old code here worked the same albeit being much simpler